tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc

```diff
@@ -165,7 +163,7 @@ public class CustomerForm extends FormLayout {
 
 In this part, we add functionality to update and delete `Customer` objects.  
 
-. In `CustomerForm.java` add buttons as Java fields: 
+. In `CustomerForm.java`, add buttons as Java fields: 
```

Comment:
The original text is probably written by me but the Java fields still sounds bit weird to me. Let's change that into class variables while we are on it.
```suggestion
. In `CustomerForm.java`, add buttons as class variables: 
```

Reply:
I'll fix this